package.json

```diff
@@ -2,7 +2,7 @@
   "license": "Apache-2.0",
   "private": true,
   "engines": {
-    "node": ">=18.0.0"
+    "node": ">=18.20.0"
```

Comment:
I got a weird error about Yarn not being able to find the aliases for stellar-sdk. Upgrading to node 18.2 seemed to fix it for some reason 🤷